File: Sistema/src/app/models/CarteiraByConsultorio.Model.ts
export class CarteiraByConsultorioModel {
  carCodi: number;
  pesCodi: number;
  pesNome: string;
  docNume: string;
  pecCodi: number;
  medicoNome: string;
  tpcCodi: number;
  tpcDesc: string;
  hosCodi: number;
  hosDesc: string;
  carDtCi: Date;
  carStat: boolean;

  constructor() {
    this.carCodi = 0;
    this.pesCodi = 0;
    this.pesNome = "";
    this.docNume = "";
    this.pecCodi = 0;
    this.medicoNome = "";
    this.tpcCodi = 0;
    this.tpcDesc = "";
    this.hosCodi = 0;
    this.hosDesc = "";
    this.carDtCi = new Date;
    this.carStat = false;
  }
}

File: API/Models/CarteiraByConsultorioModel.cs
using Microsoft.Extensions.Hosting;

namespace API_BetterLife.Models
{
    public class CarteiraByConsultorioModel
    {
        public long carCodi { get; set; }
        public long pesCodi { get; set; }
        public string? pesNome { get; set; }
        public string? docNume { get; set; }
        public int pecCodi { get; set; }
        public string? medicoNome { get; set; }
        public int tpcCodi { get; set; }
        public string? tpcDesc { get; set; }
        public int hosCodi { get; set; }
        public string? hosDesc { get; set; }
        public DateTime carDtCi { get; set; }
        public bool carStat { get; set; }
    }
}
